tools/encodings/GF Glyph Sets/GF-latin-pro_optional-glyphs.nam

```diff
@@ -0,0 +1,37 @@
+          at.case
+          hyphen.case
+          endash.case
+          emdash.case
+          periodcentered.case
+          bullet.case
+          slash.case
+          backslash.case
+          parenright.case
+          bracketleft.case
+          bracketright.case
+          braceleft.case
+          braceright.case
+          guilsinglleft.case
+          guilsinglright.case
+          guillemetleft.case
+          guillemetright.case
+          0x0030 zero.sinf
```

Comment:
_components for construction_ list could be an idea for future. I also have a 'development' filter list of Cyrillic glyphs that require anchors placing prior to generating accented glyphs.